spec/index.bs

```diff
@@ -3765,17 +3765,18 @@ interface GPUQueue {
     void signal(GPUFence fence, GPUFenceValue signalValue);
 
     void writeBuffer(
-        ArrayBuffer sourceData,
-        GPUSize64 sourceOffset,
-        GPUBuffer destination,
-        GPUSize64 destinationOffset,
-        GPUSize64 size);
+        GPUBuffer buffer,
+        GPUSize64 bufferOffset,
+        GPUSize64 size,
```

Comment:
While the general idea to reorder this sound good, seeing the size in the middle is quite strange.
Perhaps, we could leave it to the end and also default to something (not sure if 0 is good) for meaning the whole array buffer?